ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committing/ProcessTransactionSubmission.scala

```diff
@@ -281,6 +274,8 @@ private[kvutils] case class ProcessTransactionSubmission(
       }),
       delay {
         Metrics.accepts.inc()
+        logger.trace(
+          s"Transaction accepted, correlationId=${txEntry.getSubmitterInfo.getCommandId}")
```

Comment:
To improve readability you might want to create a helper function for this:
```scala
import com.daml.ledger.participant.state.kvutils.CorrelationIdHelper.fromTransactionEntry
...
logger.trace(s"Transaction accepted, correlationId=${fromTransactionEntry(txEntry)}")
...
```

Reply:
I'd rather have a logger that implicitly inserts the correlationId's into the log messages so we cannot forget them.
Since we have this requirement for all committers it might be worth making it part of that interface. We'll just need a way for the `Committer` to extract the correlationId (and other information that goes into every log message) from the actual implementation.

@miklos-da since you'll likely look into porting ProcessTransactionSubmission to the new abstraction, could you rethink how we do logging in a clean way?

Reply:
Agreed on transparently adding context information to logs -- will come up with a proposal.